example/windows/example/example.vcxproj

```diff
@@ -175,6 +169,9 @@
       <SubType>Designer</SubType>
     </Page>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CppWinRT" Version="2.0.211028.7" />
```

Comment:
```suggestion
    <PackageReference Include="Microsoft.Windows.CppWinRT" Version="2.0.210312.4" />
```

Reply:
What's the reasoning behind this? This broke the build when I tried it locally :/

Reply:
ideally we roll over to the latest and greatest cppwinrt, unless there's a cross-project ABI issue